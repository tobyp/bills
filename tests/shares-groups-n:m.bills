#!./bills
ABC	DEF.	3.00
