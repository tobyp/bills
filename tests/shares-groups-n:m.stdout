todo: D pays 1.0 to C
todo: E pays 1.0 to B
todo: F pays 1.0 to A
